resolwe/flow/tests/test_templatetags.py

```diff
@@ -0,0 +1,70 @@
+# pylint: disable=missing-docstring
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import os
+import shutil
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from resolwe.flow.managers import manager
+from resolwe.flow.models import Process, Data
+
+
+class ProcessFieldsTagsTest(TestCase):
```

Comment:
Problem is that `Data` objects are hydrated in a special way before rendering (i.e. `id` becomes `__id`). I can fake that, but I've decided to run real process as this is one of the significant things that are being tested.